### ISSUE TITLE
TypeError: issubclass() arg 1 must be a class

### ISSUE THREAD

**Issue by spyoungtech** [spyoungtech/behave-webdriver#69]
There is a bug where if a transformer class is not set, this error will occur.

**Comment by spyoungtech** [comment]
The error comes from line 111 this in `transformers.py`

```python
transformer_class = context.transformer_class if 'transformer_class' in context else None
if transformer_class and (isinstance(transformer_class, partial) and issubclass(transformer_class.func, TransformerBase)) or issubclass(transformer_class, TransformerBase):
```

The original intent was that `if transformer_class ...` should have short-circuited. However, the `or` clause is still evaluated. The entire part after the first `and` should be parenthesized.

**Comment by spyoungtech** [comment]
Fixed in #70 -- v0.2.2
